conf/prod.routes

```diff
@@ -12,4 +12,6 @@ GET        /admin/metrics                             @com.kenshoo.play.metrics.
 
 POST       /email-verification/verification-requests  @uk.gov.hmrc.helptosavestub.controllers.EmailVerificationController.verify
 
-GET        /pay-as-you-earn/02.00.00/individuals/:nino        @uk.gov.hmrc.helptosavestub.controllers.PayePersonalDetailsController.getPayeDetails(nino)
\ No newline at end of file
+POST       /help-to-save/dwp-uc-check/:nino           @uk.gov.hmrc.helptosavestub.controllers.DWPController.dwpClaimantCheck(nino)
```

Comment:
this endpoint should mirror the URL that DWP are going to supply us with. Details are here: 

https://confluence.tools.tax.service.gov.uk/display/H2S/DWP+API+2c+Get+HtS+Eligibility+Check

The URL is going to look something like:
```
https://{dwpServer}/v1/isEligible/AB123456C?systemId=607&thresholdAmount=650&transactionId=2a07d761-70ea-4a10-ae00-00d4dd53dea9
```

so this entry in `prod.routes` needs to look something like:
```
POST       /v1/isEligible/:nino           @uk.gov.hmrc.helptosavestub.controllers.DWPController.dwpClaimantCheck(nino: String, systemId: String, thresholdAmount: Int, transactionId: Option[String])
```

Reply:
Ok, I have updated the end point and the controller, I guess theres no need to do anything with the extra params in the stub?